index.html

```diff
@@ -1,151 +1,252 @@
-                <li class="logo">
-                    <a href="#"><img alt="logo" src="../src/assets/images/my_logo.png"></a>
-                </li>
-                <li class="menu">
-                    <a href=""><img alt="Menu" src="../src/assets/images/img_icon.png"></a>
-                </li>
-            </ul>
-        </nav>
+      <nav class="nav-bar">
+        <ul class="list-logo">
+          <li class="logo">My Logo</li>
+        </ul>
+        <ul class="list-menu">
+          <li class="menu-icon">
+            <img src="./src/assets/images/menu-icon.svg" alt="menu icon" />
+          </li>
+          <li class="menu-item"><a href="#">Portfolio</a></li>
+          <li class="menu-item"><a href="#">About</a></li>
+          <li class="menu-item"><a href="#">Contact</a></li>
+        </ul>
+      </nav>
+      <div class="title-text">
+        <div class="reg-text">
+          <h1 class="h1-text">
+            I’m TeeJay,<br />
+            Glad to see you!
+          </h1>
+          <p>
+            I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don’t hestiate to
+            contact me.
+          </p>
+          <a class="connect" href="#">LET’S CONNECT</a>
+        </div>
+
+        <ul class="icons">
+          <li><img src="./src/assets/images/twitter.svg" alt="Twitter" /></li>
+          <li><img src="./src/assets/images/linkedin.svg" alt="LinkedIn" /></li>
+          <li><img src="./src/assets/images/medium.svg" alt="Medium" /></li>
+          <li><img src="./src/assets/images/github.svg" alt="Github" /></li>
+          <li>
+            <img src="./src/assets/images/angellist.svg" alt="Angellist" />
+          </li>
+        </ul>
+      </div>
     </header>
-    <section class="wrap">
-        <h1>I am TEEJAY! Happy to have you here today!!</h1>
-        <p>I’m a software developer! I can help you build a product ,
-            feature or website Look through some of my work and experience!
-            If you like what you see and have a project you need coded,
-            don’t hestiate to contact me.</p>
-        <a href="#">LET'S CONNECT</a>
-        <ul class="social">
-            <li><a href=""><img src="../src/assets/images/Vector.svg" alt="Facebook" width="24" height="20"></a></li>
-            <li><a href=""><img src="../src/assets/images/Linkedin.svg" alt="Linkedin" width="24" height="20"></a></li>
-            <li><a href=""><img src="../src/assets/images/Vector_1.svg" alt="Instagram" width="24" height="20"></a></li>
-            <li><a href=""><img src="../src/assets/images/Vector_2.svg" alt="Twitter" width="24" height="20"></a></li>
-            <li><a href=""><img src="../src/assets/images/Vector_3.svg" alt="Tiktok" width="24" height="20"></a></li>
+    <section class="main-grid">
+      <!-- Card 1 -->
+      <div class="cards">
```

Comment:
Kindly, as the previous reviewer recommended, I recommend you use the Article tag instead of "div class='card'"